frontend/layouts/project-page/component.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Do we need this layout? The project page is a public page; there's no authentication nor permission checking required.